docs/tutorials/guides/mouse-input.mdx

```diff
@@ -0,0 +1,190 @@
+		Width += (uint)(currMousePos.X - prevMousePos.X);
+	}
+
+	this.prevMouseState = currMouseState;
+
+	base.OnUpdate(frameTime);
+}
+```
+
+This code checks if the left mouse button is currently being held down. If it is, then the width of the window will be
+changed based on the difference between the current and previous mouse positions.  As you move the mouse left and right,
+it will take the difference between the position on the X-axis and apply that to the width of the window.
+
+The important part here is the `this.prevMouseState = currMouseState`.  As long as the logic is added between the current
+and previous mouse states, there is something to compare to know if the mouse has moved.
+
+### <span class="color-sub-step">Step 3.3: Changing the window height</span>
+
+Now we can add the code to change the height of the window as we scroll the mouse wheel.
+Add the following code right below the `if` block added in the previous step.
+
+```csharp
+private void OnUpdate(FrameTime frameTime)
+{
+	... 
+
+	// Get the current value of the mouse scroll wheel
+	var wheelValue = currMouseState.GetScrollWheelValue();
+
+	// If the mouse wheel was not scrolled, exit the method
+	if (wheelValue != 0)
+	{
+		// Check if the mouse wheel was scrolled down or up.
+		// If the value is negative, the mouse wheel has been scrolled in the down direction.
+		var scrolledDown = wheelValue < 0;
+
+		// Change the height of the window based on the direction the mouse wheel was scrolled.
+		Height = scrolledDown
+			? Height - HeightChangeAmount
+			: Height + HeightChangeAmount;
+	}
+
+	this.prevMouseState = currMouseState;
+
+	base.OnUpdate(frameTime);
+}
+```
+
+This code will get the current value of the mouse wheel and use this to detect which direction the mouse wheel was scrolled.
+When the mouse wheel is scrolled down toward you, it returns a negative value and when it is scrolled up away from you,
+it returns a positive value.
+
+If the mouse wheel has not been scrolled at all for the current frame, it will be 0.  Only change the height if the mouse
+has been scrolled.
+
+If the mouse was scrolled in either direction, increase or decrease the height of the window.
+That's it!!  Now we are controlling the width and height of the window using the mouse!!
+This of course can be used for _**ANYTHING**_.  Character movement, aiming a weapon, and game menu item selection
+some examples of what the mouse can be used for in a game.
+
+
+### <span class="color-sub-step">Step 3.4: Run it</span>
+
+Run the application and you should be able to change the width and height of the window using the mouse.
+To change the width of the window, hold down the left mouse button and move the mouse from left to right.
+To change the height of the window, scroll the mouse wheel up or down.
+
+## <span class="color-step">Step 4: Bonus</span>
+
+It would be nice to see the current size of the window. To do this, we can override the `OnResize()` method
```

Comment:
Up to you if you wanna reword this
```suggestion
It would be nice to see the current size of the window, right? To do that, we can override the `OnResize()` method
```